### ISSUE TITLE
[android] Switch ETA minutes with ETA time on tap in navigation mode

### ISSUE THREAD

**Issue by biodranik** [organicmaps/organicmaps#4000]
Now estimated minutes are bigger than the estimated time of arrival. Some users with bad eyesight prefer to see the bigger time.

The best solution would be to draw both times in large, visible fonts if they fit into smaller devices' screen widths.
But if it doesn't fit, or as a simper workaround, times may switch on tap (it is important to save what is displayed bigger now in this case).

**Comment by Aryan-Sagavekar** [comment]
![Screenshot_20230303-191546_Organic Maps (Debug)](https://user-images.githubusercontent.com/109228835/222736328-cb00631d-ea0f-4ca6-947e-8ef635e19028.jpg)
![Screenshot_20230303-191557_Organic Maps (Debug)](https://user-images.githubusercontent.com/109228835/222736338-d61263de-60c6-4812-ab6c-b48752fc5f3d.jpg)
Is this the way you expect the Eta to be? When tapped on the time it changes to just minutes.

**Comment by biodranik** [comment]
No, that's something different, and it doesn't look user-friendly.

**Comment by Aryan-Sagavekar** [comment]
Okay @biodranik , I think this should be it. I tried fitting both the time values in same size but it did not look good. so it changes Estimated minutes to ETA on click. looks good?
![Screenshot_20230304-091010_Organic Maps (Debug)](https://user-images.githubusercontent.com/109228835/222874115-c19ff2b9-6688-403b-ac8e-b5b82549d244.jpg)
![Screenshot_20230304-091004_Organic Maps (Debug)](https://user-images.githubusercontent.com/109228835/222874124-87a8dd9f-34b7-4d3b-9d8b-48b184bc5c3c.jpg)

**Comment by biodranik** [comment]
1. The current display of ETA on the master branch and in stores is OK and should not be changed.
2. A tap handler should be added to the displayed ETA.
3. That tap handler should switch displayed values but save the existing formatting and colors. E.g. before the tap: 4h 22m on the top, 23:12 on the bottom with a smaller font. After the tap: 23:12 on the top, 4h 22m with a smaller font at the bottom.

P.S. I see that the location arrow on your screenshots is covered by the panel. It would be great if you (or someone else) will fix it by finishing/testing #3462 and #3441
